integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkEnvIT.java

```diff
@@ -32,18 +30,20 @@ public abstract class SparkEnvIT extends SparkUtilIT {
   private static final Logger LOG = LoggerFactory.getLogger(SparkEnvIT.class);
   private static final ContainerSuite containerSuite = ContainerSuite.getInstance();
 
-  protected FileSystem hdfs;
   private final String metalakeName = "test";
-
   private SparkSession sparkSession;
-  private String hiveMetastoreUri = "thrift://127.0.0.1:9083";
   private String gravitinoUri = "http://127.0.0.1:8090";
-  private String warehouse;
+
+  protected String hiveMetastoreUri = "thrift://127.0.0.1:9083";
```

Comment:
could you place `protected` variable  before `private` ?

Reply:
i would like to use `hiveMetastoreUri` in `SparkIcebergCatalogHiveBackendIT`.

Reply:
done